kitchen/ph5toms

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-#   Version: 2016.133
-EXEC='ph5tomsAPI'
+#   Version: 2017.167
+EXEC='ph5toms'
```

Comment:
Maybe this should be _ph5tosacms_, or something like that, since it is used to convert to both miniseed and sac.